src/plugins/data-card/data-card-tile.tsx

```diff
@@ -41,12 +47,14 @@ export const DataCardToolComponent: React.FC<ITileProps> = observer(function Dat
   const displaySingle = !content.selectedSortAttributeId;
   const shouldShowAddField = !readOnly && isTileSelected && displaySingle;
   const attrIdsNames = content.existingAttributesWithNames();
-  const cardOf = `Card ${content.caseIndex + 1 } of `;
+  const cardOf = `Card ${content.caseIndexNumber + 1 } of `;
 
   // When a highlighted case or cell is set, show it
-  const selectedCaseId = dataSet.firstSelectedCaseId ? dataSet.firstSelectedCaseId : dataSet.firstSelectedCell?.caseId;
+  const selectedCaseId = dataSet.firstSelectedCaseId !== undefined
+    ? dataSet.firstSelectedCaseId : dataSet.firstSelectedCell?.caseId;
   useEffect(() => {
-    if (selectedCaseId) {
+    if (content.caseIndex === undefined) return;
```

Comment:
A comment here should be added explaining why we are bailing out of this useEffect.